crates/iota-single-node-benchmark/src/mock_storage.rs

```diff
@@ -53,11 +57,11 @@ impl InMemoryObjectStore {
     // functions. (similarly the one in simulacrum)
     pub(crate) fn read_objects_for_execution(
         &self,
-        shared_locks: &dyn GetSharedLocks,
+        epoch_store: &Arc<AuthorityPerEpochStore>,
```

Comment:
Any reason not to take `&AuthorityPerEpochStore` instead?

Reply:
That's how it was done in the upstream.

Reply:
I know, but we don't have to blindly copy their mistakes